src/DataGrid.tsx

```diff
@@ -300,6 +318,14 @@ function DataGrid<R, K extends keyof R, SR>({
     return eventBus.subscribe('SELECT_ROW', handleRowSelectionChange);
   }, [eventBus, onSelectedRowsChange, rows, rowKey, selectedRows]);
 
+  useEffect(() => {
+    return eventBus.subscribe('SELECT_CELL', selectCell);
+  });
+
+  useEffect(() => {
+    latestDraggedOverRowIdx.current = draggedOverRowIdx;
```

Comment:
We are using `setDraggedOverRowIdx={isDragging ? setDraggedOverRowIdx : undefined}`. When the drag starts we rerender all the rows but on subsequent render we only render the row that fires `onMouseEnter`. If we add a setter then all the rows will be rerendered during dragging, We can use `useCallback` to prevent that. What do you prefer?

Reply:
I'd prefer `useCallback` over `useEffect` for this. I feel like `useEffect` could cause rare timing issues.